### PR TITLE
[DUOS-2723] Update Policy Pages

### PR DESCRIPTION
**Pull request by hams7504**
### Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2723

### Summary

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
👍🏽

File: src/pages/DMSPolicyInfo.jsx [hunk 1]
```diff
@@ -189,15 +192,23 @@ function DMSPolicyInfo(props) {
...
-                  leverage DUOS&apos; internal DAC for a fee.
-                </p>
+
+                {
+                  variant === 'NIH' ?
+                    <p>
+                      In <b>DUOS</b>, institutions are able to set up their own central DAC, allow investigators to create and administer DACs themselves, or
+                      leverage DUOS&apos; internal DAC for a fee.
+                    </p> :
+                    <p><b>DUOS</b> can register open-access and controlled-access datasets, regardless of the storage locationn, or if the DAC received
```
Comment (fboulnois) [inline src/pages/DMSPolicyInfo.jsx]:
```suggestion
                    <p><b>DUOS</b> can register open-access and controlled-access datasets, regardless of the storage location, or if the DAC received
```

**Review by fboulnois — CHANGES_REQUESTED** [review]
lgtm, one recommendation:

File: src/pages/DMSPolicyInfo.jsx (hunk 1 above)
Comment (hams7504) [reply src/pages/DMSPolicyInfo.jsx]: Thanks for catching that!

**Review by fboulnois — APPROVED** [review]
👍

**Pull request merged**